package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@emerson-eps/prettier-config",
-  "version": "0.0.0-development",
+  "version": "1.0.4",
```

Comment:
Not needed - this is handled by `semantic-release`

Reply:
I agree that this is not required, but semantic-release does not update it, so it is a bit disturbing when reading the package.json (not a big deal).
I can reset this if we'd rather not touch it.
This PR is mostly to actually trigger a release due to previous bad commit messages :/

Thanks for the feedback

Reply:
No problem. If you really want it you can configure it - see https://semantic-release.gitbook.io/semantic-release/support/faq#why-is-the-package.jsons-version-not-updated-in-my-repository